packages/atomic/src/components/facets/facet-search/facet-search.tsx

```diff
@@ -67,6 +68,9 @@ export class FacetSearch {
       onSubmit: () => props.controller.facetSearch.search(),
       onSelectValue: (element) => {
         const index = (element as HTMLLIElement).value;
+        if (index === FacetSearch.ShowMoreResults) {
```

Comment:
I don't think it is clean to use the `onSelectValue` function. Showing more results is a different interaction to selecting a value. Couldn't it have a dedicated function?

Reply:
I don't think it would be possible to do it outside of here, in order for the down arrow to work naturally, the show more results button also has to be a list element, and the interaction with that list is handled by the combobox.

So just to clarify `onSelectValue` here is a callback when a list element inside the combobox is selected (which includes the facet search values and the show more button). What I was checking is that if the selected element is the show more button we call `facetSearch.showMore()` otherwise it's a facet search value and we call `facetSearch.select(value)`.

Reply:
Hmm, I'm not sure if this is good though, or worth refactoring to be able to do. From an accessibility perspective, shouldn't the `show more/show less` be buttons rather than list items? Using an `<li>` does not feel correct,